index.html

```diff
@@ -56,10 +61,10 @@
         </style>
     </head>
     <body>
-        <nav class="menu">
+        <nav class="menu js-menu">
             <ul>
-                <li><a href="#home">Home</a></li>
-                <li><a href="#subway_map">Subway Map (available offline)</a></li>
+                <li><a tabindex="0" href="#home">Home</a></li>
+                <li><a tabindex="0" href="#subway_map">Subway Map (available offline)</a></li>
```

Comment:
This looks good 👍 

I just noticed the `tabindex="0"`. Is this necessary? Or does it function the same without it?

Reply:
You are right! I deleted the tabindex and everything works fine. Tabindex is making the 'untabable' elements, like<div>, into the tab order. Since <a> is naturally in the tab order and programmatically focusable, this is redundant here.